### ISSUE TITLE
Add support for WashTower

### ISSUE THREAD

**Issue by mdisabato** [nVuln/homebridge-lg-thinq#18]
**Which device do you want to add? Please describe:**
LG WashTower Model #WKGX201HBSS (Spec sheet attached)

**Describe the solution you'd like:**
Monitor operational cycles for washer and dryer as two separate devices. 

**Device Data:**

[5/30/2021, 8:07:03 AM] [LGThinQ] Found device:  769328ba-01d9-1b11-815c-f8b95a3561c0: Washer (undefined F3S1CWK__)
[5/30/2021, 8:07:03 AM] [LGThinQ] data:  {"deviceId":"769328ba-01d9-1b11-815c-f8b95a3561c0","deviceType":221,"modelName":"F3S1CWK__","subModelNm":null,"sensorType":null,"alias":"Washer","deviceCode":"LA07","networkType":"02","tftYn":"N","guideTypeYn":"Y","guideType":"TYPE1","pccModelYn":"Y","autoOrderYn":"N","drServiceYn":"N","ssid":"Pergola","timezoneCode":"America/New_York","timezoneCodeAlias":"USA/New_York","sdsGuide":"","newRegYn":"N","remoteControlType":null,"fareTarget":null,"area":"334505","sleep":null,"deviceState":"E","rmsClientId":null,"regDtUtc":"20210416191928000","regIndex":0,"blackboxYn":"Y","groupableYn":"Y","controllableYn":"N","combinedProductYn":"Y","masterYn":"N","snapshot":{"washerDryer":{"rinseDefault":"AUTODETECT_MANUAL","smartCare_onOff":"SMARTCARE_BUTTON_OFF","initialTimeMinute":0,"preState":"INITIAL","dnn_temp":"20","drumLight":"DRUMLIGHT_OFF","washTime":"NO_WASHTIME","addGarment":"ADDGARMENT_OFF","dnn_precipitationProbability":"40","dnn_washingIndex":"75","autoSoak":"AUTOSOAK_OFF","dnnReady":"DNNREADY_OFF","preWash":"PREWASH_OFF","state":"POWEROFF","ezCSSoftenerSetVal":"EZCSSO_OFF","intensive":"INTENSIVE_OFF","wrinkleCare":"WRINKLECARE_OFF","downloadCourse":"RINSESPIN","smallUE":"SMALLUE_OFF","buzzer":"BUZZER_4","AIDDLed":"AIDDLed_OFF","cloudCourse":"NOT_SELECTED","dnn_humidity":"87","freshCare":"FRESHCARE_OFF","extraRinse":"EXTRARINSE_OFF","medicRinse":"MEDICRINSE_OFF","ezDispenseNotation":"ezDisNotation_ml","turboWash":"TURBOWASH_OFF","remainTimeMinute":1,"steamSoftener":"STEAMSOFTENER_OFF","warmWater":"WARMWATERT_OFF","delay":"DELAY_OFF","soak":"NO_SOAK","childLock":"CHILDLOCK_OFF","remoteStart":"REMOTE_START_OFF","rinseHold":"RINSE_HOLD_OFF","audibleSDS":"AUDIBLE_SDS_OFF","ezDispenseDrawerState":"ezDisDrawer_CLOSE","favorite":"FAVORITE_OFF","rinseSpin":"RINSE_SPIN_OFF","courseSpendWater":0,"autoDetection":"AUTO_DETECTION_OFF","laundryTexture":0,"standby":"STANDBY_OFF","timeSave":"TIMESAVE_OFF","speechRecognitionMode":"SPEECH_RECOGNITION_MODE_OFF","courseSpendPower":0,"error":"ERROR_NO","dryLevel":"NO_DRYLEVEL","ezDetergentState":"EZCSDT_NOT_EMPTY","washLoadDisplay":"WASHLOAD_DISPLAY_OFF","creaseCare":"CREASECARE_OFF","ezSoftenerAmount":0,"rinseCount":"NO_RINSE","course":"NOT_SELECTED","protocolVersion":0,"steam":"STEAM_OFF","sterilize":"STERILIZE_OFF","multiStain":"MULTISTAIN_OFF","rinse":"RINSE_NORMAL","temp":"NO_TEMP","doorLock":"DOORLOCK_OFF","reserveTimeMinute":0,"ezCSDetergentSetVal":"EZCSDT_OFF","loadItem":"LOADITEM_OFF","washingIndex":"75","baseDownloadCourseData":"NOT_SELECTED","TCLCount":8,"waterPlus":"WATERPLUS_OFF","smartGridEnable":"SMARTGRIDENABLE_OFF","fabricSoftener":"FABRIC_SOFTENER_OFF","saveEnergy":"SAVE_ENERGY_OFF","wifiSDS":"WIFI_SDS_OFF","ezSoftenerState":"EZCSSO_NOT_EMPTY","coolDown":"COOLDOWN_OFF","dnn_dust":"-","ecoHybrid":"ECOHYBRID_OFF","voiceState":"VOICESTATE_OFF","doorClose":"DOORCLOSE_OFF","spin":"NO_SPIN","ezDetergentAmount":0,"waterLevel":"WATERLEVEL_0","soilWash":"NO_SOILWASH","activeStandbyEnable":"AUTO_STANDBY_OFF","coldWash":"COLDWASH_OFF","loadLevel":"LOAD_AUTO_DETECT"},"mid":287202642,"online":true,"static":{"deviceType":"221","countryCode":"US"},"meta":{"allDeviceInfoUpdate":true,"messageId":"39lPipYtT4-hjajUtoCfyQ"},"timestamp":1622377577189},"manufacture":null,"online":true,"platformType":"thinq2","homeDeviceOrder":-1,"roomDeviceOrder":-1,"ownershipYn":"Y","modelJsonVer":"9.4","modelJsonUri":"https://objectcontent.lgthinq.com/b6e6e5f8-6289-4b09-873c-c007cea50288?hdnts=exp=1673490225~hmac=62b97250d0476e4bfdaf3785b08d53e1c7622bb0c35eb363d4f05c295c5a7c7b","appModuleVer":"3.51","appModuleUri":"https://objectcontent.lgthinq.com/62b9bb3d-f174-458a-b4b3-e30c7ba14a31?hdnts=exp=1684561803~hmac=4bffa2e0617b8faa0635532676e9e0b55277ccc841a5f2d65a3dbd476738f721","appRestartYn":"Y","appModuleSize":"5258056","langPackProductTypeVer":"80.5","langPackProductTypeUri":"https://objectcontent.lgthinq.com/a7e610e1-1ca3-4545-bc04-5c61ad83254b?hdnts=exp=1685083049~hmac=b366db5d6f9c4e01219e8303d6ebdaf71ecc0e9dad83d2afc087202caf378a18","langPackModelVer":"0.3","langPackModelUri":"https://objectstore.lgthinq.com/28fdac4b-755d-4c78-b901-54e5e521d7ff?Expires=1661645814&Signature=W1wGgOPcQ4vvjdTreBMNwHSfGC7QT92muGntFxi~XkNzUuI5PCyYSmcyI2nO27~A9FagJ9veaSF1dTkUcbUQlFEAqTlKcgkoxi5Zm9kZsXMNkFUb2XYNkiwUem4UCywIfKUG681jCEunzkfXGu9sh4yIyrLqKVb3NupEVo67-q1Y7uFzWbjdz~hQSov5~UKOFAMnJd5zUE4ks~PTXRyLLGvk6dnWT0HtGftYwKo88Pwjdjcr2xNJwk4G5jI7LJfYNdnEmIGq3t92nmbi-iHt3jamZC6U8AdehQRH-DHZ585uE~5yy5AKCzOr-4fRgHpZcecHgAtSZUdGEhjZew~MWA__&Key-Pair-Id=APKAI74R6YENXPGRIWLQ","roomId":"161859762791865546","fwInfoList":[{"checksum":"0000585A","partNumber":"SAA42102401","order":2},{"checksum":"0000A415","partNumber":"SAA42102301","order":1}],"modemInfo":{"modelName":"F3S1CWK__","appVersion":"clip_hna_v1.9.080","modemType":"QCOM_QCA4010","ruleEngine":"y"},"existsEntryPopup":"N","fwVer":null,"modemVer":null,"subDeviceCount":0,"firebaseLogKey":"T:LC-S:LC-U:4.5-P:KEPLER"}
[5/30/2021, 8:07:03 AM] [LGThinQ] Device not supported: 769328ba-01d9-1b11-815c-f8b95a3561c0: Washer (undefined F3S1CWK__)
[5/30/2021, 8:07:03 AM] [LGThinQ] Found device:  83f7c886-e876-125f-b299-f8b95a356217: WashTower (undefined W3S1CVK2EVS_D_US)
[5/30/2021, 8:07:03 AM] [LGThinQ] data:  {"deviceId":"83f7c886-e876-125f-b299-f8b95a356217","deviceType":222,"modelName":"W3S1CVK2EVS_D_US","subModelNm":null,"sensorType":null,"alias":"WashTower","deviceCode":"LA06","networkType":"02","tftYn":"N","guideTypeYn":"Y","guideType":"TYPE1","pccModelYn":"Y","autoOrderYn":"N","drServiceYn":"N","ssid":"Pergola","timezoneCode":"America/New_York","timezoneCodeAlias":"USA/New_York","sdsGuide":"","newRegYn":"N","remoteControlType":null,"fareTarget":null,"area":"334505","sleep":null,"deviceState":"E","rmsClientId":null,"regDtUtc":"20210416191928000","regIndex":0,"blackboxYn":"Y","groupableYn":"Y","controllableYn":"N","combinedProductYn":"Y","masterYn":"Y","snapshot":{"washerDryer":{"initialBit":"INITIAL_BIT_OFF","smartCare_onOff":"SMARTCARE_BUTTON_OFF","initialTimeMinute":1,"preState":"INITIAL","downloadedCourseDryer27inchBase":"PERMPRESS","energySaver":"ENERGYSAVER_OFF","error":"ERROR_NO","dryLevel":"NO_DRYLEVEL","moreLessTime":0,"dnnReady":"DNNREADY_OFF","smartCourseDryer27inchBase":"NOT_SELECTED","remainTimeHour":0,"state":"POWEROFF","reduceStatic":"REDUCESTATIC_OFF","wrinkleCare":"WRINKLECARE_OFF","temp":"NO_TEMP","antiBacterial":"ANTIBACTERIAL_OFF","loadItem":"LOADITEM_OFF","AIDDLed":"AIDDLed_OFF","cloudCourse":"NOT_SELECTED","dampDrySignal":"DAMPDRYSIGNAL_OFF","timeDry":"TIMEDRY_OFF","remainTimeMinute":1,"courseDryer27inchBase":"NOT_SELECTED","easyIron":"EASYIRON_OFF","childLock":"CHILDLOCK_OFF","remoteStart":"REMOTE_START_OFF","turboSteam":"TURBOSTEAM_OFF","initialTimeHour":0},"mid":350167405,"online":true,"static":{"deviceType":"222","countryCode":"US"},"meta":{"allDeviceInfoUpdate":true,"messageId":"eCidgBMhT7ycrhGVp6yA7g"},"timestamp":1622377634350},"manufacture":null,"online":true,"platformType":"thinq2","homeDeviceOrder":-1,"roomDeviceOrder":-1,"ownershipYn":"Y","modelJsonVer":"3.0","modelJsonUri":"https://objectcontent.lgthinq.com/d3b8a3d0-3019-4229-a4ff-122ecf948a6f?hdnts=exp=1675409957~hmac=202ba30119caa4f769151e24abccc3b5ee52d3cd554b00d8042607ffb2adcccf","appModuleVer":"3.51","appModuleUri":"https://objectcontent.lgthinq.com/62b9bb3d-f174-458a-b4b3-e30c7ba14a31?hdnts=exp=1684561803~hmac=4bffa2e0617b8faa0635532676e9e0b55277ccc841a5f2d65a3dbd476738f721","appRestartYn":"Y","appModuleSize":"5258056","langPackProductTypeVer":"80.5","langPackProductTypeUri":"https://objectcontent.lgthinq.com/a7e610e1-1ca3-4545-bc04-5c61ad83254b?hdnts=exp=1685083049~hmac=b366db5d6f9c4e01219e8303d6ebdaf71ecc0e9dad83d2afc087202caf378a18","langPackModelVer":"0.4","langPackModelUri":"https://objectstore.lgthinq.com/ae5670c7-4c73-4f81-a05a-cbd06cc841cc?Expires=1661646940&Signature=KY~vyBNY-RKA9ICugMCdG3qNw1LuxmKmen3UU7~VHTrfZErUp9R6xPz~mZO0gyoCw22rl00cgbeM9oS9pfCl0cLlcE33~KK4eMj4Jh5hB0LKs6nBrgXZrU1Dff1v63HljKQRpyiGwOUIdIecOwkgI4EU7u0cvqWxcYjTHI9MVcIsSgbrhV1d5kiBkPrxGZcHixtDMuuV4oXVEHwiquo-xt-f5ykafVkkgBrdSmn1dINoxHhtULZSNL-3jaatHHlvXuVtAT8ruyk7nGdVUAN8yGvoviLqw2je8dLs3MZ03Uq86x0TQjHmHKKjOFh6bT8WJlBt6LmgdbfFoVQZovOkLw__&Key-Pair-Id=APKAI74R6YENXPGRIWLQ","roomId":"161859762791865546","fwInfoList":[{"checksum":"0000B7B2","partNumber":"SAA38964419","order":2},{"checksum":"0000FFDB","partNumber":"SAA38964313","order":1}],"modemInfo":{"modelName":"W3S1CVK2EVS_D_US","appVersion":"clip_hna_v1.9.080","modemType":"QCOM_QCA4010","ruleEngine":"y"},"existsEntryPopup":"N","fwVer":null,"modemVer":null,"subDeviceCount":0,"firebaseLogKey":"T:LC-S:LC-U:4.5-P:KEPLER"}
[5/30/2021, 8:07:03 AM] [LGThinQ] Device not supported: 83f7c886-e876-125f-b299-f8b95a356217: WashTower (undefined W3S1CVK2EVS_D_US)

[WashTower Spec Sheet.pdf](https://github.com/nVuln/homebridge-lg-thinq/files/6688715/WashTower.Spec.Sheet.pdf)

**Comment by nVuln** [comment]
I already added, it's work for you ?

**Comment by mdisabato** [comment]
Hi!

Just after I posted the request, the WashTower started leaking from the main feed valve. The retailer took it away, and we just received the replacement the other day. We ran a few loads through it to make sure it works, and I monitored the plug in. I’ll send you logs and screen caps next week after I do some controlled testing. So far it seems to work well. There are a few labels that might be changed, and I’ll let you know what they are. 

Thank you for all your hard work on this!

Michael
